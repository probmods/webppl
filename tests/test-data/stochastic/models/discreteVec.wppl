var model = function() {
  return sample(Discrete({ps: Vector([1, 3])}));
};
