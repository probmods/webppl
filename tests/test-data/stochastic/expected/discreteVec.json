{
  "hist": {
    "0": 0.25,
    "1": 0.75
  }
}
